nornir_nautobot/plugins/tasks/dispatcher/default.py

```diff
@@ -507,6 +510,34 @@ class NetmikoDefault(DispatcherMixin):
 
     config_command = None  # This can be removed in future versions, as it is not used in the base class.
     offline_commands = False
+    netmiko_kwargs = {}
+
+    @classmethod
+    def _get_netmiko_kwargs(cls, obj) -> dict:
+        """
+        Retrieves Netmiko keyword arguments from various sources with a class attribute fallback.
+
+        Order of precedence:
+            1. Custom field 'netmiko_kwargs' (string, then dict)
+            2. Config context 'netmiko_kwargs' (string, then dict)
+            3. Class default `cls.netmiko_kwargs`
+        """
+        sources = [
+            obj.cf.get("netmiko_kwargs"),
+            obj.get_config_context().get("netmiko_kwargs"),
+        ]
+
+        for source in sources:
+            if isinstance(source, dict):
+                return source
+            if isinstance(source, str):
+                try:
+                    return json.loads(source)
+                except json.JSONDecodeError:
+                    # Fall through to the next source if JSON parsing fails
+                    pass
+
+        return cls.netmiko_kwargs
```

Comment:
Do we want to add the arista eos fix to the class definition?

Reply:
do we want to manage it?  I don't think we want to be the owners of it. But thats my two cents.  We're giving them 3 ways to inject what they want now.